pkg/rpc/adminservice/base.go

```diff
@@ -135,8 +138,17 @@ func NewAdminServer(kubeConfig, master string) *AdminService {
 	scheduledWorkflowExecutor := workflowScheduler.GetWorkflowExecutor(executionManager, launchPlanManager)
 	logger.Info(context.Background(), "Successfully initialized a new scheduled workflow executor")
 	go func() {
+		logger.Info(context.Background(), "Starting the scheduled workflow executor")
 		scheduledWorkflowExecutor.Run()
```

Comment:
do you want to remove this one?

Reply:
I wanted to support the default where no reconnect attempts are specified, but I can if you think this is cleaner